homeassistant/components/ollama/strings.json

```diff
@@ -25,7 +25,8 @@
       "init": {
         "data": {
           "prompt": "Prompt template",
-          "max_history": "Max history messages"
+          "max_history": "Max history messages",
+          "keep_alive": "Duration in seconds for Ollama to keep model in memory. -1 = indefinite, 0 = never."
```

Comment:
Additional instructions should go into the `data_description` section (not `data` which are just plain field names).